.github/workflows/release.yml

```
@@ -0,0 +1,35 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - 'main'
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+      - '[0-9]+.[0-9]+.[0-9]+rc[0-9]+'
+
+jobs:
+  release-scalarflow-charts:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Install Helm
+        uses: azure/setup-helm@v3
+
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.5.0
+        with:
+          charts_dir: scalarflow/charts
+        env:
+          CR_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
          CR_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          CR_SKIP_EXISTING: true
```
If we publish the unstable/development version chart based on the `main` branch, we have to override the development version chart in every commit for the `main` branch.

So, we have to allow overriding the existing chart (already released chart).

However, if we don't publish the unstable/development version chart, we can set this setting `CR_SKIP_EXISTING: true`.

By using this configuration, we restrict overriding existing charts. In other words, we can avoid overriding the existing stable version chart by accident.

---

Thanks for correcting, I applied it